### ISSUE TITLE
[Quest][TBC] Master of Elixirs - Non available quest

### ISSUE THREAD

**Issue by schoefel** [dalaranwow/dalaran-wow#5444]
[//]: # (INFORMATION ABOUT TAGS)
[//]: # (The black and yellow tag is unique. There can only be one label assigned to them - one black and one yellow)
[//]: # (Priority - ... tags are assigned by team members)
[//]: # (Your issue will be reviewed by testers and the developers will fix it in the future.)
[//]: # (Only once the issue got the label [Fixed - On Live Server] you can find the fix on your Gameserver.)
[//]: # (You can increase the relevance of your issue by adding a thumb-up emote http://i.imgur.com/RFouVVi.png)

[//]: # (***************************************************************)
[//]: # (** !!!!!!!!!!!!!DON'T DELETE THIS TEMPLATE ELSE YOUR ISSUE WILL BE CLOSED!!!!!!!!!!!! **)
[//]: # (***************************************************************)

**Current behaviour**: Tell us what happens in detail (Where, When, What, Why, Who)
Lorokeem doesn't give any quests
**Expected behaviour**: Tell us what should happen instead
He is supposed to give Master of Elixirs once you are lvl 68 and Alchemy 325
**Steps to reproduce the problem**:

1. Get Alchemy up
2. become lvl 68
3. try finding the quest

**Include proofs for this behaviour**

You can research this on several pages like http://wowhead.com or http://wowwiki.wikia.com or http://www.youtube.com/
Issues without proofs will be closed and tag On Hold will be added until the proof is provided.

**Include the ID for the game objects, npcs (creatures,pets,minions), spells, items, quests, instances, zones, achievements, skills**
Master of Elixirs: https://wotlk.evowow.com/?quest=10902
Lorokeem: https://wotlk.evowow.com/?npc=19052
You can research this on several pages like https://wowgaming.altervista.org/aowow/ or https://wowhead.com/

**Include Screenshots from the issue if necessary**

**Comment by Rushor** [comment]
This quest works fine on PTR.

![GitHub Logo](https://i.imgur.com/OEFOGdc.jpg)

You will need to complete prevquest https://wowgaming.altervista.org/aowow/?quest=10906 + have skill 171 (Alchemy) at skillpoint: 325
